resources/roca_main.brs

```diff
@@ -26,9 +24,31 @@ sub main()
     }
 
     for each file in files
-        path = ["pkg:", basePath, file].join("/")
-        suite = _brs_.runInScope(path, args)
+        suite = _brs_.runInScope(file, args)
         args.index += 1
     end for
 end sub
 
+' Recursively searches directories for '*.test.brs' files starting at `path`.
+'
+' @param {string} [path="pkg:"] - the path to search
+' @param {roArray} [testFiles=[]] - the current set of discovered test files
+'
+' @returns {roArray} an array containing the fully-qualified path to each '*.test.brs' file
+'                    discovered recursively from `path`
+function __roca_findTestFiles(path = "pkg:" as string, testFiles = [] as object) as object
+    files = ListDir(path)
+
+    for each f in files
+        if f.inStr(".") = -1
```

Comment:
Is this just an optimization to not recurse on files? This seems fine, but I think directories can have periods in them.  Might be better to omit the check or use a different check list ListDir where you only recurse if it has files.  That would omit files, but also omit empty directories.